images/sdn/Dockerfile.rhel

```diff
@@ -15,7 +15,7 @@ RUN INSTALL_PKGS=" \
       libmnl libnetfilter_conntrack conntrack-tools \
       libnfnetlink iproute procps-ng openssl \
       iputils binutils xz util-linux dbus nftables \
-      " && \
+      tcpdump conntrack-tools" && \
```

Comment:
`conntrack-tools` is already listed above. (kube-proxy depends on it)

Reply:
Good catch, removed.